web/jest.config.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  rootDir: 'src',
+  roots: ['<rootDir>', '<rootDir>/../../api-client'],
+  verbose: true,
+  automock: false,
+  setupFilesAfterEnv: [
+    '<rootDir>/../jest.setup.js'
+  ],
+  transformIgnorePatterns: [
+    'node_modules/(?!build-configs|api-client)'
+  ],
+  moduleNameMapper: {
+    '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': '<rootDir>/__mocks__/fileMock.js',
+    '\\.(css)$': 'identity-obj-proxy',
+    hashids: 'hashids/dist/hashids.min.js'
+  },
+  moduleFileExtensions: [
+    'js',
+    'json'
+  ],
+  moduleDirectories: [
+    'node_modules'
+  ],
+  coverageDirectory: '<rootDir>/../reports/coverage',
```

Comment:
so there is a shared coverage directory for web and native? would it make sense to separate it to `'<rootDir>/../reports/web/coverage'` and `'<rootDir>/../reports/native/coverage'`?

Reply:
No there is no shared directory. rootDir is src/